Add documentation for deploying ChromeOS test extensions. (#12579)
fleetdm/fleet: docs/Contributing/README.md

### [API for contributors](./API-for-contributors.md) 
Get to grips with Fleet API routes. This documentation is helpful for developing or contributing to Fleet.

### [Deploying ChromeOS Test Extension](./Deploying-chrome-test-ext.md) 

> Learn how to force deploy a test Chrome extension to ChromeOS devices for debug purposes.

Reviewer: ```suggestion
Learn how to deploy a test version of the fleetd Chrome extension for debug purposes.
```
